vanilla-framework/vanilla-framework
# Slider input

## Done

Widened the input in sliders to allow for up to 4 digits

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/slider/slider-input/
- Change one of the slider inputs to type="number"
- Resize the window to mobile
- Change the slider's "max" attribute to 9999
- Increase the value to 9999
- See that all numbers are visible

Fixes #1926
